app/src/App/routes/SurgeRoutes.tsx

```diff
@@ -1351,7 +1351,7 @@ function DeploymentCatalogueNavigate() {
             'international-disaster-response-law': surgeCatalogueOtherInternationalDisasterResponseLaw,
             migration: surgeCatalogueOtherMigration,
             'national-society-development': surgeCatalogueOtherNationalSocietyDevelopment,
-            'partnership-and-resource-development': surgeCatalogueOtherPartnershipResourceDevelopment,
+            'partnership-and-resource-development': surgeCatalogueOtherStrategicPartnershipsResourceMobilisation,
```

Comment:
shouldn't we also change the URL to `partnership-and-resource-mobilisation`

Reply:
Thanks Sameer, good catch! Did it...